tests/test_generate_fasta.py

```diff
@@ -5,33 +5,121 @@
 from subprocess import call
 from filecmp import cmp
 import py_compile
+from pvacseq.lib import generate_fasta
 
-class GenerateFastaTests(unittest.TestCase):
-    def setUp(self):
-        self.python = sys.executable
+class GenerateVariantSequences(unittest.TestCase):
```

Comment:
Not sure if this is an unintentional error made when rebasing, but the test case name should reflect the new name of the module